circle.yml

```diff
@@ -0,0 +1,61 @@
+machine:
+  environment:
+    DOCKER_IMAGE_NAME: prom/memcached_exporter
```

Comment:
If I'm not mistaken, `_` are not allowed in docker image names. That's why we have `node-exporter` instead of `node_exporter`